docs/userguides/networks.md

```diff
@@ -1,6 +1,82 @@
 # Networks
 
-When interacting with the blockchain, you will have to select a network.
+When interacting with a blockchain, you will have to select an ecosystem (e.g. Ethereum, Arbitrum, or Fantom), a network (e.g. Mainnet or Goelri) and a provider (e.g. Eth-Tester, Geth, or Alchemy).
+Networks are part of ecosystems and typically defined in plugins.
+For example, the `ape-ethereum` plugin comes with Ape and can be used for handling EVM-like behavior.
+
+## L2 Networks
+
+Common L2 networks, such as Arbitrum, Polygon, Optimism, or Fantom, have ApeWorX-maintained (trusted) plugins that override the Ethereum ecosystem API class and change any defaults that are needed.
+You can install these plugins by doing:
+
+```shell
+ape plugins install arbitrum polygon optimism fantom
+```
+
+Each plugin does different things.
+In general, L2 plugins are very small and override the Ethereum ecosystem class.
+Here are some examples of changes L2 plugins make that allow improved support for these networks:
+
+1. Networks that don't support EIP-1559 transactions use Static-fee transaction types by default whereas `ape-ethereum` will use EIP-1559 transactions by default.
+2. Some networks, such as `ape-arbitrum`, have unique transaction types (and receipt types!) that are handled in the plugin.
+   This logic does not have to live in the base `ape-ethereum` plugin but can live in the network's custom plugin.
+3. Fee token information: When displaying gas reports or other data, network plugins can use the correct fee-token symbols, such as Polygon MATIC.
+
+Here is a list of all L2 network plugins supported by Ape:
```

Comment:
Wonder if there's a way to link to this list in github using search and tags on our repos

Reply:
ya ive been thinking about the tags thing for a long time.
we should have a way to see what plugins do what and show that in `ape plugins list`.